srcs/server/src/auth/exceptions/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./badCredentials.exception";
+export * from "./shouldBeLogged-in.exception";
+export * from "./internalAuth.exception";
+export * from "./userAlreadyRegistered.exception";
+export * from "./credentialsTaken.exception";
```

Comment:
I like this idea of having an index for exception to ease the import. I will apply it on my code.